src/augur-node/subscribe-to-event.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+var augurNodeState = require("./state");
+var submitJsonRpcRequest = require("./submit-json-rpc-request");
+
+function subscribeToEvent(eventName, params, callback) {
+  params = Array.isArray(params) ? params : [];
+  params.unshift(eventName);
+
+  submitJsonRpcRequest("subscribe", params, function (err, response) {
+    if (err) throw new Error("Error subscribing to event: " + err);
+    augurNodeState.setCallback("event:" + response.subscription, callback);
```

Comment:
It might be cleaner to have separate `set` and `get` functions for subscriptions, vs always doing string concatenation with `"event:"`.